### DISCUSSION TITLE
Phase 2 Processing Time and CPU Utilization Phenomenon

### DISCUSSION THREAD

**Discussion by actuaryck** [Chia-Network/chia-blockchain#3146] [Q&A]
I am running on a Ryzen 5 3600 16GB RAM machine and plotting on a RAID-0 storage composed of three 7200rpm HDD. 

For my first attempt I ran just a single plot (k-32), each step of Phase 2 took 2,910s on average to complete and the average CPU utilization was 9.71%.

After finishing this plot, I started two plots simultaneously. For each step for Phase 2, the average time taken and CPU utilization for these two plots are 1,075s and 80% and 5,588s and 7.38%.

I totally cannot understand why when running two plots together one plot can complete so much quicker than the other plot. It is even quicker than my first attemp when only one plot was running. Any suggestion on why this happens?

![Phase2_TimeTaken](https://user-images.githubusercontent.com/83434425/116657835-a8c1fd80-a9c1-11eb-8d1c-1021828c2966.JPG)

**Comment by droidomz** [comment]
This is expected behavior just add a delay for second  plot to start when running in parallel.
